### DISCUSSION TITLE
Fans completely off on shutdown?

### DISCUSSION THREAD

**Discussion by steve8x8** [OpenNeptune3D/OpenNept4une#93] [Q&A]
Is it possible to put the printer at complete rest on shutdown, i.e. halt the remaining fans? (I suppose the mainboard fan can be software-controlled; I don't know how "smart" the PSU fan is/can be made.)
Since the screen is still powered after a shutdown, can the overall state be shown there (at least for a certain while if there's any screen timeout)?

**Comment by Justdigit** [comment]
I had a silent fan mod on the old printer.cfg before a did a fresh install.
It was only cooling the motherboard when necessary. And not with each minor move.

This evening i am going to try to adapt the old. Printer.Cfg to the newer printer.cfg. There are things changed.

**Reply by halfmanbear** [reply]
The issue isn't the host chip or the MCU overheating, it's the stepper motor drivers on the mainboard (no temperature monitoring). 

Unfortunately the hotend heatbreak cooling fan and the mainboard fan are wired in parallel. 

With Elegoos configuration these stepper drivers don't get any cooling during moves where the hotend isn't being heated. (Full bed meshing, homing / tramming etc) therefore I set the mainboard fan to come on when the stepper motors are activated. I've configured  a 15 min cool-off timer, although, you can most likely reduce this value significantly without issue although I need to do some more testing.

**Comment by halfmanbear** [comment]
> Is it possible to put the printer at complete rest on shutdown, i.e. halt the remaining fans? (I suppose the mainboard fan can be software-controlled; I don't know how "smart" the PSU fan is/can be made.) Since the screen is still powered after a shutdown, can the overall state be shown there (at least for a certain while if there's any screen timeout)?

Running a poweroff command on the Linux host simply, powers down the rockchip64 part of the board. The MCU side & all the auxiliary systems (steppers etc) are still powered, even though they reside on the same board. 

Unfortunately it was intentionally designed to be turned off at the mains (not good practice). 

With my current understanding there is no way to easily poweroff the PSU via software. 

Ideally if you would like to shut the system down remotely.. 

Issue a poweroff command via terminal /  fluidd Web UI (or could even configure a poweroff macro using the gcode_shell_command plugin)

Then toggle a smart plug connected to the printer.

**Reply by Justdigit** [reply]
Oh but the PSU Fan is not the issue here.... For me not anymore at least.

I habe used this solution... I did the same with the Neptune2 works great.

https://youtu.be/LNyZnfy3bRE
